lib/ember-cli/asset_resolver.rb

```diff
@@ -0,0 +1,53 @@
+module EmberCLI
+  class AssetResolver
+    def initialize(options)
```

Comment:
You can follow along in https://rubygems.org/gems/ember-cli-rails to check downloads per version. Most people are on latest 0.3.5. While we are on `0.` we have freedom to decide on dependencies, if we need and wnat to.